export_parse.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+import argparse
+import pandas as pd
+from os import makedirs, getcwd
+from os.path import exists, join
+from shutil import copy
+import sys
+
+def export_parse(input_file, source_path, out_path, failed):
+    """
+    Takes in an export csv from cellswiper, filter out either the passing or failing annotations,
+    and copies those files to the out_path
+    """
+    try:
+        annotations = pd.read_csv(input_file)
+    except:
+        print("Could not read input file")
+        sys.exit(1)
+    
+    if not exists(source_path):
+        print("Source path for images does not exist")
+        sys.exit(1)
+
+    # convert from relative to absolute path
+    out_path = join(getcwd(), out_path)
+    filtered = annotations[annotations['annotation'] == ('f' if failed else 't')]
+
+    try:
+        if not exists(out_path):
+            makedirs(out_path)
+    except:
+        print("Could not create out_path " + out_path)
+        sys.exit(1)
+
+    failures = 0
+
+    for img in filtered['image_name']:
+        try:
+            copy(join(source_path, img), join(out_path, img))
+        except FileNotFoundError:
+            print("Could not find " + img)
+            failures += 1
+            continue
+    
+    print("Copy complete")
+    print(str(len(filtered) - failures) + " successes, " + str(failures) + " failures")

```

Comment:
Maybe use `str.format` here to make this cleaner without having to cast everything